analysis/README.rst

```diff
@@ -0,0 +1,44 @@
+Scenario Player Analysis
+========================
+This is an analysis tool to show performance metrics of a scenario using the logs of a scenario run.
+
+Prerequisites
+"""""""""""""
+- `pyenv <https://github.com/pyenv/pyenv>`_
+- `virtualenvwrapper <https://github.com/virajkanwade/venvwrapper>`_
+
+You may also use other python virtual environments
+
+Installation
+""""""""""""
+
+.. code-block:: bash
+
+    pyenv shell 3.7.0
+    mkvenv raiden-analysis
+    pip install -r requirements.txt
+
+How To Use
+""""""""""
+Either start the analysis directly or print out the help:
+
+.. code-block:: bash
+
+    python analysis.py $PATH_TO_SCENARIO_LOG_FILE
+
+    python analysis.py -h
+
+Result
+""""""
+There are currently two output files generated:
+
+- a gantt plot is rendered in a HTML file called (default: ``raiden-gantt-analysis.html``)
```

Comment:
```suggestion
- a gantt plot is rendered in an HTML file called (default: ``raiden-gantt-analysis.html``)
```

Reply:
That would be wrong then ;-)

Reply:
Actually not. You spell it after how you'd pronounce it. And "H" is pronounced with an "a" sound in the beginning, so the spelling should actually be what Christian suggests.

Reply:
Sorry - almost forgot about that complexity